### ISSUE TITLE
NVDA is announcing "Menu" text twice after activating "Right Click to bring Menu" button.

### ISSUE THREAD

**Issue by msftedad** [nvaccess/nvda#13208]
![Announcing menu text twice](https://user-images.githubusercontent.com/93735775/147784150-55e407df-df1e-499d-a694-7535d25d1cb0.png)

### Steps to reproduce:
1.	Extract the provided sample project that is “Request to log external Bug to NVDA BUG 10700596” on your system.
2.	Open test project "ContexMenu_20211208.zip".
3.	Open “Bin” file.
4.	Open “ContexMenu” application.
5.	TAB to "Right Click to bring Menu" button.
6.	Activate "Right Click to bring Menu" button by using ALT+ SHIFT + F10.
7.	Start Screen reader NVDA and observe the announcement.

### Actual behavior:
TAB to "Right Click to bring Menu" button and activate the button using ALT + SHIFT + F10 to navigate the “Menu”. NVDA has been announcing "Menu" text twice after the expansion of the “Menu”.

### Expected behavior:
NVDA should announce "Menu" text only once after activating "Right Click to bring Menu" button.

### System configuration
Application: Azure RTOS - Trace X 

#### NVDA installed/portable/running from source:
Installed

#### NVDA version:
NVDA(2021.3)

#### Windows version:
Win11

#### Does the issue still occur after restarting your computer?
Yes

#### Have you tried any other versions of NVDA? If so, please report their behaviors.
Yes, Same with NVDA2021.3.1

#### Does the issue still occur after you run the COM Registration Fixing Tool in NVDA's tools menu?

[Request to log external Bug to 'NVDA'  BUG 10700596.zip](https://github.com/nvaccess/nvda/files/7810083/Request.to.log.external.Bug.to.NVDA.BUG.10700596.zip)

**Comment by feerrenrut** [comment]
The sample has not been provided. Please include the source code (so that we can inspect) and instructions for us to build it locally.

**Comment by msftedad** [comment]
I have updated the sample application for the issue.

**Comment by Adriani90** [comment]
@msftedad the application attached is not working, probably it is built on a very old .net core framework.
Are you still having this issue with NVDA 2024.2 Beta?

**Comment by msftedad** [comment]
Hi @Adriani90 , This issue is closed at our end.If we face issue further we will log a new one.Thanks

**Comment by Adriani90** [comment]
Thanks for reporting, closing this one then.
